guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-server.adoc

```diff
@@ -0,0 +1,55 @@
+[id="Renewing_a_Custom_SSL_Certificate_on_Server_{context}"]
+= Renewing a {customssltitle} certificate on {ProjectServer}
+
+[NOTE]
+====
+If {ProjectServer} is already configured with CA-signed SSL certificates and has been renewed by the same Certificate Authority, you do not have to replace the consumer package `katello-ca-consumer-latest.noarch.rpm` on clients registered to {ProjectServer}.
+
+If the validity of CA-signed SSL certificates installed on {Project} from Certificate Authority A expires, and you installed an SSL certificate from Certificate Authority B, then you must re-deploy the `katello-ca-consumer-latest.noarch.rpm` package on every client registered to {Project}.
+This is because you require a new CA certificate for a client to authenticate or communicate with {Project}.
+====
+
+Use this procedure to update your renewed {customssl} certificate signed by a Certificate Authority (CA) for {ProjectServer}.
+The `katello-certs-check` command validates the input certificate files and returns the commands necessary to renew a {customssl} certificate for {ProjectServer}.
+
+.Prerequisite
+Refer to the https://access.redhat.com/documentation/en-us/red_hat_satellite/6.13/html-single/installing_satellite_server_in_a_connected_network_environment/index#Configuring_Server_with_a_Custom_SSL_Certificate_satellite[Configuring {ProjectServer} with a Custom SSL Certificate] guide before creating a new CSR because the Server certificate should have X.509 v3 `Key Usage` and `Extended Key Usage` extensions with required values.
```

Comment:
```suggestion
ifdef::satellite[]
.Prerequisite
Refer to the https://access.redhat.com/documentation/en-us/red_hat_satellite/6.13/html-single/installing_satellite_server_in_a_connected_network_environment/index#Configuring_Server_with_a_Custom_SSL_Certificate_satellite[Configuring {ProjectServer} with a Custom SSL Certificate] guide before creating a new CSR because the Server certificate should have X.509 v3 `Key Usage` and `Extended Key Usage` extensions with required values.
endif::[]
```

and we have an attribute for links to access.redhat.com: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L7 (also with the version as attribute)

Reply:
Undoubtedly :+1: